### PR TITLE
Feature/trucks add complete trip

### PR DESCRIPTION
**Pull request by justaname94**
review #14 first

### CODE REVIEW

File: trucks/functions/update-truck/main.go [hunk 1]
```diff
@@ -51,6 +51,14 @@ func apiGatewayHandler(ctx context.Context, request events.APIGatewayProxyReques
 		truck.Year = year
 	}
 
+	if body.Get("completed_trips") != "" {
```
Comment (pablocampogo) [inline trucks/functions/update-truck/main.go:54]: I think this action should not be legal, we are just let it increment naturally

File: trucks/functions/increment-completed-trips/main.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	if err != nil {
+		return apigateway.LogAndReturnError(err), nil
+	}
+
+	err = storage.IncrementTruckCompletedTrips(ctx, truck.TruckID)
+	if err != nil {
+		return apigateway.LogAndReturnError(err), nil
+	}
+
+	return apigateway.NewJSONResponse(200, nil), nil
```
Comment (pablocampogo) [inline trucks/functions/increment-completed-trips/main.go:34]: We should return a message or something here "Successfully incremented" or we could even return the whole truck structure

**Review by pablocampogo — CHANGES_REQUESTED** [review]
Solve comments here and in the dependency

File: trucks/functions/update-truck/main.go (hunk 1 above)
Comment (justaname94) [reply trucks/functions/update-truck/main.go:54]: so you mean to remove this (and only letting the change occur in the event that increments the value) or only allow sequential increment?

**Review by pablocampogo — APPROVED** [review]

**Pull request merged**